sentry_sdk/integrations/aws_lambda.py

```diff
@@ -116,6 +116,13 @@ def sentry_handler(event, context, *args, **kwargs):
                     )
                     hub.capture_event(event, hint=hint)
                     reraise(*exc_info)
+                finally:
```

Comment:
@untitaker In this "finally" block, the code to stop the thread basically works in case of :-
1) A successful run of lambda function.
2) When an exception is there in the code.
But, it doesn't stop the thread in case of timeout, which is an expected behavior.
By "stop the thread" I mean explicitly stopping a thread by sending a stop signal.